config.xml

```diff
@@ -42,8 +42,8 @@
         <variable name="ANDROID_HOST" value=" " />
         <variable name="ANDROID_PATHPREFIX" value="/" />
     </plugin>
-    <plugin name="cordova-plugin-airgap-secure-storage" spec="git+ssh://git@github.com:aeternity/cordova-plugin-airgap-secure-storage.git#master" />
+    <plugin name="cordova-plugin-airgap-secure-storage" spec="~1.0.0" />
```

Comment:
In what public registry, I can find this package?
https://www.npmjs.com/package/cordova-plugin-airgap-secure-storage

Reply:
@davidyuk it was made automatically by cordova! not sure why.